### DISCUSSION TITLE
Why does iOS14+ require jailbreak?

### DISCUSSION THREAD

**Discussion by divinity76** [utmapp/UTM#2519] [UTM Help (iOS)] [answered]

**Comment by conath** [comment, marked as answer]
As of iOS 14.6, Apple does not provide an API for running virtual machines on iOS. UTM used to rely on a bug in iOS 13 and 14.3 that allowed VMs to run anyway, using a technology called JIT (simplifying here).

Since iOS 14.4 this loophole has been closed. Therefore it is no longer possible to run UTM without either of the other two ways of enabling JIT on iOS: jailbreak or debugger from a computer. 

An alternative way of running VMs is in the works called UTM-SE, however its performance is reduced compared to JIT and it will not be available on the App Store (because Apple chooses to disallow UTM there).
